openg2p/models/openg2p_beneficiary_api.py

```diff
@@ -0,0 +1,265 @@
+from odoo import fields
+from odoo.http import Controller, route, request
+
+
+class Openg2pBeneficiaryApi(Controller):
+    # categories
+    @route("/program-categories", type="json", auth="user", methods=["GET"])
+    def categories(self):
+        try:
+            category_list = request.env["openg2p.program.enrollment_category"].search(
+                []
+            )
+            res = []
+            for category in category_list:
+                res.append(category.api_json())
+            return {
+                "status": 200,
+                "message": "Success",
+                "count": len(res),
+                "categories": res,
+            }
+        except BaseException as e:
+            return {"status": 200, "error": str(e)}
```

Comment:
Status shouldn't be 200 in case of error

Reply:
Changed status to 500